packages/framework/src/Resources/config/services.yml

```
@@ -524,7 +524,9 @@ services:
 
     # Define RedirectController explicitly to prevent creating it on-the-fly
     # when redirecting (it causes unnecessary container rebuild).
+    # ContainerControllerResolver fetch Controller from container, therefore it must be a public service
```

Since you are so explicit about class names you should do it properly
```suggestion
    # ContainerControllerResolver fetch RedirectController from container, therefore it must be a public service
```